packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.tsx

```diff
@@ -34,6 +34,15 @@ type TState = {
   hoveredRowKey: number;
 };
 
+function formatValue(value: number) {
+  if (value < 0.1) {
+    return `< 0.1 req/s`;
+  }
+
+  // Truncate number to two decimal places
+  return `${value.toString().match(/^-?\d+(?:\.\d{0,2})?/)![0]} req/s`;
```

Comment:
Nit: this is not so much a question of decimal places, but of precision. E.g. 2.666667 is just as useless (too much precision) as 26666.67, but this code will allow the latter, whereas I would prefer to represent it as 26.6k - never more than 3 digits of precision

Reply:
Good point and totally accepted. However, will be better just to go this path right now, and just improve on another iteration. Your comment definitely enhances the solution, but the current step will already provide an improvement which is immediate.

Reply:
[yurishkuro](https://github.com/yurishkuro) ?